pkgs/tools/filesystems/grive2/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, pkgconfig, fetchurl, yajl, cmake, libgcrypt, curl, expat, boost, binutils }:
+
+stdenv.mkDerivation rec {
+  version = "0.5.0";
+  name = "grive2-${version}";
+
+  src = fetchurl {
+    url = https://github.com/vitalif/grive2/archive/v0.5.0.tar.gz;
+    md5 = "f291ffaef831abf3605417995082bcc3";
+  };
+
+  buildInputs = [cmake pkgconfig libgcrypt yajl curl expat stdenv binutils boost];
```

Comment:
`stdenv` and `binutils` are redundant `buildInputs`; also, strictly build-time dependencies such as `cmake` and `pkgconfig` can be moved to `nativeBuildInputs`.